docs/docs/references/applications/README.mdx

```diff
@@ -0,0 +1,67 @@
+import ApplicationTypesIntro from '../../fragments/_application_types_intro.mdx';
+
+# 📱 Applications
+
+## Introduction
+
+Each real-world application that requires user authentication using Logto needs to be registered as an application in Logto.
+
+## Application ID
+
+_Application ID_ is a unique auto-generated identifier to identify your application with the Logto application.
+
+## Application Types
+
+We provide three application types to help developers integrate their applications with Logto:
+
+<ApplicationTypesIntro />
+
+## Application Settings
+
+### Application Name
+
+_Application Name_ is a human-readable name of the application and will be displayed in the admin console.
+
+### Description
+
+A brief description of the application will be displayed on the admin console application details page.
+
+### Authorization Endpoint
+
+_Authorization Endpoint_ is an OIDC term, and it is used to interact with the Logto service and obtain an authorization grant.
+
+In Logto, The authorization endpoint is uneditable. It redirects users to the sign-in page.
+
+You can check out the [Authorization Endpoint](https://openid.net/specs/openid-connect-core-1_0.html#AuthorizationEndpoint) for more information.
+
+### Redirect URIs
+
+Allowed URIs that redirect the user to the application after signing in to Logto.
```

Comment:
should specify will a search param or fragment be recognized?

Reply:
I claim this in the following line:
> It will be used to validate the redirect URI used in the authorization request, so every redirect URI to be used should be pre-added to this allowed list.

Developers will likely use our SDKs, and that makes developers hard to see the search parameters.
I think it's ok to describe it as "the redirect URI used in the authorization request.".